low-level/framework/sspl_rabbitmq_reinit

```diff
@@ -257,8 +257,8 @@ def _create_vhost_if_necessary(virtual_host, product):
             response = _send_command(command)
 
         vhosts = response.split('\n')
-        assert vhosts[0] == 'Listing vhosts ...'
         for vhost in vhosts[1:-1]:
+            print(vhosts)
```

Comment:
this would print same 'vhosts' in loop, i think you meant 'vhost'. Or just print vhosts before loop, better

Reply:
That vhosts output will be difficult to understand if gets printed directly..S, printing it within a loop only but will be displayed as:
vhost: <Name of vhost>

Reply:
same vhosts will be printed multiple times, better before loop then.

Reply:
I think I changed the approach now in the latest patch.